pages/GSoC.md

```diff
@@ -341,6 +341,17 @@ should be opened in DispVM or locally)
 
 **Mentor**:
 
+### New Qubes Manager
+**Project**: New Qubes Manager
```

Comment:
I'd remove this one, as (if everything goes according to plan...) at the time of GSoC, it should be already completed.